templates/fine-tune-llm_v2/cookbooks/customize_initial_weights_and_prompt_format/README.md

```diff
@@ -0,0 +1,172 @@
+# Customize initial weights and prompt format
+**⏱️ Time to complete**: 60 minutes
+
+This guide will showcase how you can finetune a model with a similar architecture to the Llama or Mistral family and customize the chat template/ prompt format. We will focus on fine-tuning the [Meta Llama Guard 2 model](https://llama.meta.com/docs/model-cards-and-prompt-formats/meta-llama-guard-2/) throughput this cookbook. 
+
+The two capabilities showcased here are
+1. Bringing your own weights - (1) weights of other models similar in architecture to the Llama or Mistral family of models or (2) weights from a previous finetuning run. While we focus on (1) here, (2) is an important use-case of multi-step fine-tuning covered in depth in the cookbook [here](../continue_from_checkpoint/).
+2. Customizing the chat template/ prompt format - Specify a custom prompt format for formatting input messages to easily fine-tune on _any_ data format.
+
+The Anyscale platform is uniquely suited to support both of these use-cases. This guide assumes you have familiarized yourself with the [basic fine-tuning guide](../../README.md).
+
+
+# Table of Contents
+1. [Bring your own weights](#bring-your-own-weights)
+    - [Bring models of the same architecture](#bring-models-of-the-same-architecture)
+        - [Example YAML](#example-yaml)
+        - [How do I configure access to my weights in remote storage??](#)
+            - [How do I bring my weights to Anyscale?](#how-do-I-bring-my-weights-to-Anyscale-?)
+    - [Bring checkpoints from a previous finetuning run](#bring-checkpoints-from-a-previous-finetuning-run)
+2. [Customizing the prompt format (chat template)](#customizing-the-prompt-format)
+    - [How prompt formatting works in `llmforge`](#how-prompt-formatting-works-in-llmforge)
+    - [Customizing data preprocessing and the prompt format](#customizing-data-preprocessing-and-the-prompt-format)
+
+# Bring your own weights
+
+In general, you can customize the initial weights in your fine-tuning run through two options in the YAML:
+- `initial_base_model_ckpt_path` : Path to the base model weights you wish to start with
+- `initial_adapter_model_ckpt_path`: Path to the adapter (LoRA) weights you wish to start with
+
+## Bring models of the same architecture
+You can fine-tune a model similar in architecture to the Llama or Mistral family of models to fine-tune on the Anyscale Platform. For example, to fine-tune Llama Guard 2, you can 
+specify the model ID and `initial_base_model_ckpt_path` as below:
+### Example YAML
+
+```yaml
+model_id: meta-llama/Meta-Llama-3-8B
+initial_base_model_ckpt_path: s3://my-bucket/llama-guard-2
```

Comment:
does it have to be s3 or can you get it from hf?

Reply:
Only S3 for now.